packages/input-output-logger/__tests__/index.js

```diff
@@ -18,4 +18,19 @@ describe('📦 Middleware Input Output Logger', () => {
     expect(logger).toHaveBeenCalledWith({ event: { foo: 'bar', fuu: 'baz' } })
     expect(logger).toHaveBeenCalledWith({ response: { message: 'hello world' } })
   })
+  test('It should exclude paths', async () => {
+    const logger = jest.fn()
+
+    const handler = middy((event, context, cb) => {
+      cb(null, { message: 'hello world', bar: 'bi' })
```

Comment:
what happens if `response` is not an object (e.g. a string). Should we add an extra test to check for these edge cases?

Reply:
Done [here](https://github.com/middyjs/middy/pull/507/commits/d3b0fc04503b529ee1929ef6cb2e5bcf58948a7c)

Reply:
Tnx